indico/modules/events/editing/schemas.py

```diff
@@ -218,3 +224,21 @@ def _check_if_can_unset_or_delete(self, publishable):
                            .has_rows())
             if is_last:
                 raise ValidationError(_('Cannot unset the only publishable file type'))
+
+
+class EditingReviewConditionArgs(mm.Schema):
+    class Meta:
+        rh_context = ('event',)
+
+    file_types = fields.List(fields.Int(), required=True, validate=not_empty)
+
+    @validates('file_types')
+    def _validate_file_types(self, file_types):
+        event = self.context['event']
+        event_file_types = {ft.id for ft in event.editing_file_types}
+        if set(file_types) - event_file_types:
+            raise ValidationError(_('Invalid file type used'))
+
+        event_conditions = editing_settings.get(event, 'review_conditions')
+        if file_types in event_conditions.values():
```

Comment:
Doesn't this depend on the order in which you added the conditions? Or are we sorting them somewhere?